python/lsst/daf/butler/cli/cmd/_remove_runs.py

```diff
@@ -85,5 +118,11 @@ def remove_runs(**kwargs):
         else:
             print(abortedMsg)
     else:
+        # if the user opted out of confirmation but there are runs with
+        # parent collections then they must confirm; print a message
+        # and exit.
+        if any(run.parents for run in result.runs) and not force:
+            _print_requires_confirmation(result.runs, result.datasets)
+            sys.exit(1)
```

Comment:
Is Click okay with calling `sys.exit()`? I see there is a `click.Context.exit()`.